webrtc-stats.html

```diff
@@ -430,8 +430,8 @@
             <dl data-link-for="RTCRTPStreamStats" data-dfn-for="RTCRTPStreamStats" class=
             "dictionary-members">
               <dt>
-                <dfn><code>ssrc</code></dfn> of type <span class="idlMemberType"><a>unsigned long
-                </a></span>
+                <dfn><code>ssrc</code></dfn> of type <span class="idlMemberType"><a>unsigned
+                long</a></span>
```

Comment:
nit: what's our line width limit? 100? "unsigned long" makes it 96 wide, that should be OK? Line 556 is 97 wide.